test/functional/feature_assumeutxo.py

```diff
@@ -133,13 +133,20 @@ def expected_error(msg):
         cases = [
             # (content, offset, wrong_hash, custom_message)
             [b"\xff" * 32, 0, "7d52155c9a9fdc4525b637ef6170568e5dad6fabd0b1fdbb9432010b8453095b", None],  # wrong outpoint hash
-            [(2).to_bytes(1, "little"), 32, None, "Bad snapshot data after deserializing 1 coins."],  # wrong txid coins count
+            [(2).to_bytes(1, "little"), 32, None, "Bad snapshot data after deserializing 1 coins"],  # wrong txid coins count
             [b"\xfd\xff\xff", 32, None, "Mismatch in coins count in snapshot metadata and actual snapshot data"],  # txid coins count exceeds coins left
             [b"\x01", 33, "9f4d897031ab8547665b4153317ae2fdbf0130c7840b66427ebc48b881cb80ad", None],  # wrong outpoint index
             [b"\x81", 34, "3da966ba9826fb6d2604260e01607b55ba44e1a5de298606b08704bc62570ea8", None],  # wrong coin code VARINT
             [b"\x80", 34, "091e893b3ccb4334378709578025356c8bcb0a623f37c7c4e493133c988648e5", None],  # another wrong coin code
             [b"\x84\x58", 34, None, "Bad snapshot data after deserializing 0 coins"],  # wrong coin case with height 364 and coinbase 0
-            [b"\xCA\xD2\x8F\x5A", 39, None, "Bad snapshot data after deserializing 0 coins - bad tx out value"],  # Amount exceeds MAX_MONEY
+            [
+                # VARINT(Compressed(MAX_MONEY + 1)) + VARINT(0)
+                b"\xa0\xc8\xad\xb1\xd1\x83\xff\x01" + b"\x00",
```

Comment:
as the above comment has already asked , I can't seem to understand why `VARINT(0)` was added considering we have already exceeded the max amount in the first amount written itself.

Reply:
See https://github.com/bitcoin/bitcoin/pull/31907/commits/dedc2d9cecccb866230c07112d00288237d12eaa#r1978352110